lib/crudry_resolver.ex

```diff
@@ -96,8 +98,16 @@ defmodule Crudry.Resolver do
 
       iex> Crudry.Resolver.default except: [:get!, :list, :delete]
       :ok
+
+      iex> Crudry.Resolver.default list_opts: %{order_by: :id}
+      :ok
   """
   defmacro default(opts) do
+    case opts[:list_opts] do
```

Comment:
I'm not sure this is needed, I think this can be done like the other opts:

`Module.put_attribute(__CALLER__.module, :list_opts, list_opts)`

Reply:
It can't because the list_opt is a map, and when is parsed into defmacro it turns out in as: `{:%{}, [line: 19], [order_by: :id]}` for example. Also we need to put_attribute as a map , e.g: ` %{order_by: :id}`, I can't see another way to accomplished that.